src/NotepadNext/docks/LuaConsoleDock.cpp

```diff
@@ -215,6 +215,8 @@ LuaConsoleDock::LuaConsoleDock(LuaState *l, QWidget *parent) :
         else {
             input->braceHighlight(INVALID_POSITION, INVALID_POSITION);
         }
+
+        Q_UNUSED(flags);
```

Comment:
Move this to the top of the lambda. Thats just kind of the way I've been doing it in other places so that will just help keep things consistent.

Reply:
Removing the flag/variable name will have the same effect (I am in new phase, in which I want to reduce Qt macros). See new commit.

Reply:
Personally I've always used the `Q_UNUSED` macro rather than removing the variable name (technically you can also remove the entire parameter, but I don't like that idea).

I'm not against removing the variable name, but for sake of consistency let's keep the macro. If for some reason later its decided to forego the macro route, then some other option like you suggested can be applied across the entire code base.

Even if something is a little ugly, I'd rather lean towards the side of being explicit.

Reply:
done. building and in 10 minutes we should get a green light.

Reply:
> but for sake of consistency let's keep the macro

I'm not seeing this in the latest changes.

Reply:
arg.. internet issues (rebased + squashed now): https://github.com/dail8859/NotepadNext/pull/257/files#diff-806bf892a96cd6a0af53c1c14f08bd10b2733e4cd12e71c26b998509a3f7971dR194